src/main/java/com/fauna/client/RequestBuilder.java

```diff
@@ -30,6 +31,8 @@ public class RequestBuilder {
     private static final String QUERY_PATH = "/query/1";
     private static final String STREAM_PATH = "/stream/1";
 
+    private static final int TIMEOUT_BUFFER_MS = 1000;
```

Comment:
I would suggest aligning with js here, AFAIK quite a bit of thought was given to those defaults there https://github.com/fauna/fauna-js?tab=readme-ov-file#client-timeout